lib/packwerk/file_processor.rb

```diff
@@ -56,7 +56,7 @@ def call(absolute_file)
     private
 
     sig do
-      params(node: Parser::AST::Node, absolute_file: String).returns(T::Array[UnresolvedReference])
+      params(node: ::Parser::AST::Node, absolute_file: String).returns(T::Array[UnresolvedReference])
```

Comment:
Seeing the prefixed `::` definitely makes me feel inclined to prefer `Packwerk::FileParser` over `Parser` so that we don't fight with the `parser` library. I suppose it's a "temporary" problem if we eventually swap out `parser` for `syntax_tree` or something else, but can't go wrong with `FileParser` I think.